backend/plugins/frontline_api/src/modules/integrations/call/db/models/Integrations.ts

```
@@ -0,0 +1,67 @@
+      if (!integrations) {
+        return [];
+      }
+
+      if (integrationId) {
+        const a = integrations.map((integration: ICallIntegration) => {
+          return integration.inboxId === integrationId;
+        });
+      }
+
+      const filteredIntegration = integrations.map((item: ICallIntegration) => {
+        let integration = item;
```

ESLint

[error] 'integration' is never reassigned. Use 'const' instead.

(prefer-const)